dapps/react-dapp-v2/src/chains/hedera.ts

```diff
@@ -0,0 +1,58 @@
+import { JsonRpcRequest } from "@walletconnect/jsonrpc-utils";
+
+import { HEDERA_LOGO_URL } from "../constants";
+import {
+  NamespaceMetadata,
+  ChainMetadata,
+  ChainRequestRender,
+  ChainsMap,
+} from "../helpers";
+
+export const HederaMetadata: NamespaceMetadata = {
+  testnet: {
+    logo: HEDERA_LOGO_URL,
+    rgb: "118, 90, 234",
+  },
+};
+
+export const HederaChainData: ChainsMap = {
+  testnet: {
+    name: "Hedera Testnet",
+    id: "hedera:testnet",
+    /**
+     * Hedera integration utilizes an sdk instead of a JSON-RPC relay:
+     * {@link https://specs.walletconnect.com/2.0/blockchain-rpc/hedera-rpc}
```

Comment:
This URL 404s :( Doesn't seem to be added to the specs yet?

Reply:
@bkrem won't this be added automatically once the [PR](https://github.com/WalletConnect/walletconnect-specs/pull/117) has been merged?

Reply:
I've updated the link to have the preview link until that PR is merged.